dashboard/src/components/DeploymentForm/BasicDeploymentForm/Subsection.tsx

```diff
@@ -40,11 +40,21 @@ class Subsection extends React.Component<ISubsectionProps> {
         <div
           hidden={
             param.children &&
+            !!enablerChildrenParam &&
             param.children[enablerChildrenParam] &&
             param.children[enablerChildrenParam].value !== enablerCondition
           }
         >
-          <div className="margin-v-normal">{label}</div>
+          <div className="margin-v-normal">
+            {label}
+            {param.description && (
+              <>
+                <br />
+                <span className="description">{param.description}</span>
+              </>
+            )}
```

Comment:
Out of interest, wondering why
```
{param.description && (
  <p className="description">{param.description}</p>
)}
```
wouldn't suffice, rather than a break + span?

Reply:
because when using a `<p>` the margin between lines grows too much:

![Screenshot from 2019-10-14 12-05-22](https://user-images.githubusercontent.com/4025665/66743776-f710b980-ee7a-11e9-8647-5ddc673529a3.png)

vs

![Screenshot from 2019-10-14 12-05-58](https://user-images.githubusercontent.com/4025665/66743797-0263e500-ee7b-11e9-8d4c-dfb5d70bd98c.png)

Not a very big deal but I prefer using that than change the default margin of a paragraph

Reply:
You wouldn't need to change the default margin of a paragraph though right? Just the style for p in this context. But yeah, not a big deal - I just tend to try use semantic elements rather than elements which mix semantics and presentation (ie. `<br>`).

Reply:
> You wouldn't need to change the default margin of a paragraph though right? Just the style for p in this context

Yes, sorry I meant that. I will probably send a follow up PR touching up style so maybe I can change this.